client/landing/stepper/declarative-flow/migration/index.tsx

```diff
@@ -146,6 +146,12 @@ const useCreateStepHandlers = ( navigate: Navigate< StepperStep[] >, flowObject:
 					addQueryArgs( { siteId, siteSlug, from }, SITE_MIGRATION_ASSISTED_MIGRATION.slug )
 				);
 			},
+			goBack: ( props?: ProvidedDependencies ) => {
+				const siteId = getFromPropsOrUrl( 'siteId', props );
```

Comment:
I wonder if here we could use something like the `history.back()`. So if the user mixes this button with the native back of the browser, it works properly instead of accumulating new registers in the history. WDYT?

I was thinking how to handle other cases where the back wouldn't necessarily be the previous page, and I saw we have the `replace` option in the `navigate`. Maybe we could use it for situations where we couldn't use the `history.back`.

WDYT?

Reply:
I updated to use the replace flag.

Reply:
About the`history.back` we can have the user incoming from the sign/login flow so the simple `history.back` will redirect user to wrong locations.